Build/packages.lock.json

```diff
@@ -38,9 +38,9 @@
       },
       "Speckle.InterfaceGenerator": {
         "type": "Direct",
-        "requested": "[0.9.6, )",
```

Comment:
hmm, why we downgrade it? @adamhathcock is `0.9.6` version the fix you made for namespaces lately?

Reply:
locks are messed up, trying to resolve it

Reply:
I gave up, here is a PR build on the latest dev: https://github.com/specklesystems/speckle-sharp-connectors/pull/93